plugins/connectrpc/go/v1.17.0/buf.plugin.yaml

```diff
@@ -0,0 +1,20 @@
+version: v1
+name: buf.build/connectrpc/go
+plugin_version: v1.17.0
+source_url: https://github.com/connectrpc/connect-go
+integration_guide_url: https://connectrpc.com/docs/go/getting-started
+description: Generates client and server stubs for connectrpc.com/connect. Compatible with the gRPC, gRPC-Web, and Connect RPC protocols.
+deps:
+  - plugin: buf.build/protocolbuffers/go:v1.34.2
```

Comment:
Updated this to match https://github.com/connectrpc/connect-go/blob/v1.17.0/go.mod#L13